java-client/src/main/java/org/opensearch/client/opensearch/tasks/ListRequest.java

```diff
@@ -333,7 +333,7 @@ public ListRequest build() {
 	 * Endpoint "{@code tasks.list}".
 	 */
 	public static final Endpoint<ListRequest, ListResponse, ErrorResponse> _ENDPOINT = new SimpleEndpoint<>(
-			"es/tasks.list",
+			"opensearch/tasks.list",
```

Comment:
Will removing "es/" instead of replacing have any failures? if not, i would remove it instead of replacing it.